blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java

```diff
@@ -122,10 +122,13 @@ public Boolean call() throws Exception {
                         singleBatchWriteCtx.stop();
                         bufferedMetrics.dec(batch.size());
 
-                        if (System.currentTimeMillis() - writeStartTime > timeout.toMillis()) {
+                        long now = System.currentTimeMillis();
+                        if ( now - writeStartTime > timeout.toMillis()) {
                             exceededScribeProcessingTime.mark();
-                            getLogger().error("Exceeded timeout " + timeout.toString() + " before persisting " +
-                                    "all metrics for batch " + batchId);
+                            getLogger().info(
+                                    String.format("Batch write time %d (ms) exceeded timeout %s before persisting " +
+                                            "all metrics for batch %d",
+                                            now - writeStartTime, timeout.toString(), batchId));
```

Comment:
I did look into it a little bit. I vote for changing it to debug. There is meter associated with it. I plotted it and saved on ingest dashboard. Incase we wanted to find out if ingestion is taking more than 5 seconds, we can look at that dashboard.

http://graphite00.iad.prod.bf.k1k.me/grafana/#/dashboard/db/ingest-dashboard?panelId=31&fullscreen&edit